src/docs/patterns/loading.mdx

```diff
@@ -0,0 +1,123 @@
+import { DemoCard, ImageGrid, MaterialDesignDescription, Divider } from '../../app/components';
+import Image1 from '../../app/assets/design-patterns/loading-states/loading-state-load.png';
+import Image2 from '../../app/assets/design-patterns/loading-states/loading-state.png';
+import Image3 from '../../app/assets/design-patterns/empty-states/empty-states.png';
+import Image4 from '../../app/assets/design-patterns/empty-states/empty-states-with-background-image.png';
+import InlineSpinner from '../../app/assets/design-patterns/form-validation/verify-input-on-page-change.png';
+
+<!-- KEYWORDS: design pattern loading asynchronous api refresh -->
+
+# Loading & Waiting
+
+Often times, your application will need to load data from (or send data to) a remote location with some amount of latency. When this happens, you will need to provide some indication to the user that something is happening. Different situations may call for different styles of indicator, explained below.
+
+<React.Fragment>
+    <MaterialDesignDescription
+        title={'Progress Indicators'}
+        url={'https://material.io/components/progress-indicators'}
+    />
+    <Divider />
+</React.Fragment>
+
+# Anatomy
+
+## Variations
+
+### Initial Page Loading
```

Comment:
Are we going to recommend that people use these for anything other than 'initial page load' though? This feels kind of like a unique case rather than a generic approach to loading. I might be wrong though.

Reply:
"Initial Page Loading" describes a use case, while "placeholder shapes" describes a variation.

And no, right now I couldn't think of any other place for this to show up beside the initial loading screen